spot-client/src/spot-tv/calendars/backend-calendar.js

```diff
@@ -0,0 +1,90 @@
+import { fetchCalendarEvents } from 'common/backend';
+import { logger } from 'common/logger';
+import { integrationTypes } from './constants';
+
+/**
+ * The calendar's implementation based on REST API.
+ */
+export default {
+    /**
+     * Loads the external script for accessing the backend calendar API.
+     *
+     * @param {Object} config - Values needed to properly initialize the API.
+     * @param {string} config.SERVICE_URL - The URL pointing to the backend endpoint which handles
+     * the calendar requests.
+     * @returns {Promise} Resolves when the backend API has loaded.
+     */
+    initialize(config) {
+        this.url = config.SERVICE_URL;
+
+        return Promise.resolve();
+    },
+
+    /**
+     * Requests current calendar events for a provided room.
+     *
+     * @returns {Promise<Array<Object>>}
+     */
+    getCalendar({ jwtToken }) {
+
+        logger.log(`Will fetch calendar events from: ${this.url}`);
```

Comment:
I think we should use the logger format where the second arg is an object so it's easier to query for.

Reply:
I removed this log